exercises/sublist/example.rb

```diff
@@ -0,0 +1,18 @@
+
+def sublist(set_a, set_b)
```

Comment:
Agreed, sets are unordered and the elements are unique, whereas lists are ordered and can have multiples of elements.